interface/reports/pat_ledger.php

```diff
@@ -213,7 +213,7 @@ function sel_patient() {
 if (! $_REQUEST['form_csvexport']) {
   if ( $_REQUEST['form_refresh'] && $orow <= 0) {
     echo "<span style='color:red;'>";
-    echo xlt('No data to display. Select patient and try search again.');
+    echo xlt('No data to display for this patient.');
```

Comment:
Hmmmm was this reported as a bug?

Reply:
I don't think the two statement makes any difference ie `No data to display. Select patient and try search again.` and `No data to display for this patient.`

Reply:
> I don't think the two statement makes any difference ie `No data to display. Select patient and try search again.` and `No data to display for this patient.`

The issue was user already selected a patient before performing search. So the current error message is confusing and user might keep trying to select the same patient over and over without knowing the actual cause of no results displaying. You can check the issue and comments for more clarification.

Reply:
@muarachmann This was an approved issue. It was suggested that @Ngai-E create a better error message. 
> But what I expect to happen if no billing data was found, is an appropriate response such as 'Sorry no billing data was found for this patient' instead of 'No data to display. Select Patient and try again.' meanwhile a patient was selected. User can get confused as to the actual cause of failure.